src/components/cc-pricing-header/cc-pricing-header.js

```diff
@@ -1,72 +1,113 @@
-import '../cc-flex-gap/cc-flex-gap.js';
+import '../cc-icon/cc-icon.js';
 import { css, html, LitElement } from 'lit';
 import { classMap } from 'lit/directives/class-map.js';
 import { ifDefined } from 'lit/directives/if-defined.js';
+import { iconRemixArrowDownSLine as iconArrowDown } from '../../assets/cc-remix.icons.js';
 import { dispatchCustomEvent } from '../../lib/events.js';
 import { i18n } from '../../lib/i18n.js';
 import '@shoelace-style/shoelace/dist/components/select/select.js';
-import '@shoelace-style/shoelace/dist/components/menu-item/menu-item.js';
+import '@shoelace-style/shoelace/dist/components/option/option.js';
 import { sortZones } from '../../lib/zone.js';
 import { shoelaceStyles } from '../../styles/shoelace.js';
 import { skeletonStyles } from '../../styles/skeleton.js';
 import { CcZone } from '../cc-zone/cc-zone.js';
 
-/** @type {Currency[]} */
-const SKELETON_CURRENCIES = [];
 /** @type {Zone[]} */
 const SKELETON_ZONES = [];
 
 /** @type {Currency} */
-const CURRENCY_EUR = { code: 'EUR', changeRate: 1 };
+const DEFAULT_CURRENCY = { code: 'EUR', changeRate: 1 };
+
+const DEFAULT_CURRENCY_LIST = [
+  { code: 'EUR', changeRate: 1 },
+  { code: 'GBP', changeRate: 0.88603 },
+  { code: 'USD', changeRate: 1.2091 },
+];
+
+/** @type {Temporality} */
+const DEFAULT_TEMPORALITY = { type: '30-days', digits: 2 };
+
+const DEFAULT_TEMPORALITY_LIST = [{
+  type: 'second',
+  digits: 7,
+},
+{
+  type: 'minute',
+  digits: 5,
+},
+{
+  type: 'hour',
+  digits: 3,
+},
+{
+  type: 'day',
+  digits: 2,
+},
+{
+  type: '30-days',
+  digits: 2,
+}];
 
 /**
  * @typedef {import('../common.types.js').Currency} Currency
+ * @typedef {import('../common.types.js').Temporality} Temporality
  * @typedef {import('../common.types.js').Zone} Zone
+ * @typedef {import('./cc-pricing-header.types.js').ZoneListState} ZoneListState
```

Comment:
issue: the component references `ZoneListState` but the types files is not up to date